modules/gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces_{context}"]
+= Enabling the application set resources in non-control plane namespaces
+
+As an Argo CD administrator, you can define a certain set of non-control plane namespaces wherein you can create, update, and reconcile `applicationSet` resources. You must explicitly enable and configure the Argo CD and `applicationSet` custom resources (CRs) as per your requirements.
+
+.Procedure
+
+. Set the `sourceNamespaces` parameter for the `applicationSet` spec to include the non-control plane namespaces:
++
+.Example Argo CD custom resource
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: repo
+spec:
+  applicationSet:
+    sourceNamespaces: <1>
+      - dev <2>
+----
+<1> List of non-control plane namespaces for creating and managing `applicationSet` resources in target namespaces
+<2> Target namespace for the Argo CD server to create and manage `applicationSet` resources
+
```

Comment:
Should we add a important note here with below ?
> Ensure that [Apps in Any Namespace](https://argocd-operator.readthedocs.io/en/latest/usage/apps-in-any-namespace/) is enabled on target namespace i.e the target namespace name is part of `.spec.sourceNamespaces` field in ArgoCD CR.

And we also need to mention about wildcards support. This can go as normal note or text. 
> As of now, wildcards are not supported in `.spec.applicationSet.sourceNamespaces`.

Reply:
adding this - As of now, wildcards are not supported in .spec.applicationSet.sourceNamespaces.